test/sim/moves/gmaxvolcalith.js

```diff
@@ -40,4 +40,34 @@ describe('G-Max Volcalith', function () {
 		battle.makeChoices('move sleeptalk, move sleeptalk', 'move sleeptalk, move sleeptalk');
 		assert.strictEqual(battle.p2.active[0].hp, battle.p2.active[0].maxhp - (Math.floor(battle.p2.active[0].maxhp / 6) * 4));
 	});
+
+	it.skip('should deal damage alongside Sea of Fire or G-Max Wildfire in the order those field effects were set', function () {
+		battle = common.createBattle({gameType: 'doubles'});
+		battle.setPlayer('p1', {team: [
+			{species: 'Coalossal-Gmax', item: 'Eject Button', moves: ['rockthrow', 'sleeptalk']},
+			{species: 'Charmander', moves: ['firepledge', 'sleeptalk']},
+			{species: 'Wynaut', level: 72, moves: ['grasspledge', 'nightshade', 'sleeptalk']},
+		]});
+		battle.setPlayer('p2', {team: [
+			{species: 'Roggenrola', ability: 'Receiver', item: 'Focus Sash', level: 1, moves: ['sleeptalk', 'tackle']},
+			{species: 'Boldore', ability: 'Receiver', item: 'Sitrus Berry', evs: {hp: 28}, moves: ['sleeptalk']},
+			{species: 'Torracat', ability: 'Cheek Pouch', item: 'Focus Sash', level: 1, moves: ['sleeptalk']},
+		]});
+		//Set up Volcalith first, then Sea of Fire
+		battle.makeChoices('move rockthrow 1 dynamax, move sleeptalk', 'move tackle 1, move sleeptalk');
+		battle.makeChoices('switch 3');
+		battle.makeChoices('move grasspledge 1, move firepledge 1', 'move sleeptalk, move sleeptalk');
+		battle.makeChoices('', 'switch 3');
+
+		//Bring Cheek Pouch down to Sash so it will faint and set Receiver's HP to exactly 5/8 of max
+		battle.makeChoices('move nightshade 2, move firepledge 1', 'move sleeptalk, move sleeptalk');
+		let expectedHP = battle.p2.active[1].maxhp;
+
+		/* TODO: this passes when it shouldn't because Sitrus Berry isn't triggering between Sea of Fire and G-Max Volcalith like it should;
+		 * Sitrus Berry should be able to activate in between those damaging effects. */
```

Comment:
So does this mean you want more time to work on this, or are you okay committing as-is?

Reply:
I just thought of a better test that gets around the issue. I'll toss it up in a bit.

Reply:
Never mind, my other test doesn't function how I thought it would in-game.

Reply:
Ok, came up with a new test that works much better!